### DISCUSSION TITLE
New to nvim, need help on styling nvim for rust dev

### DISCUSSION THREAD

**Discussion by PixellatteHQ** [mrcjkb/rustaceanvim#495] [Q&A]
Hi, can somebody share their nvim config for rust development for newbie like me who are migrating away from IDE to nvim? Thanks in advance

**Comment by mrcjkb** [comment]
hey 👋 

You can find lots of configurations that use rustaceanvim [here](https://dotfyle.com/neovim/configurations/top?plugins=mrcjkb/rustaceanvim).

For getting started with Neovim in general, most people recommend [kickstart.nvim](https://github.com/nvim-lua/kickstart.nvim).
